docs/en/contributing/contributing-to-docs.md

```diff
@@ -1,28 +1,256 @@
 # Contributing to documentation
 
-We use [MkDocs](https://www.mkdocs.org/) and [Github pages](https://pages.github.com/) for the documentation site you're reading right now.
+You would like to contribute to the documentation site? Great!  
+We use [MkDocs](https://www.mkdocs.org/) and [GitHub pages](https://pages.github.com/) for the documentation site you're reading right now.
 
-## Prerequisites
+Here is a step-by-step guide on how to create/edit pages.
 
-Before you begin, ensure you have the following installed on your system:
+## Step 0: Prerequisites
 
-- **Python 3** - Required for MkDocs
-- **pip** - Python package installer (usually comes with Python)
+Before you start, make sure you have the following installed on your system:
 
-Check [MkDocs' installation guide](https://www.mkdocs.org/user-guide/installation/) on how to install these dependencies.
+### Git
 
-## How to get started
+Git is required to clone the repository and manage your changes.
 
-1. Clone the `pylon-docs` repository: `git clone https://github.com/pylonmc/pylon-docs`
-2. Install all required dependencies: `pip install -r requirements.txt`
-3. Run the documentation website locally using `mkdocs serve`
+You can check if Git is already installed by running:
 
-## Deploying
+```bash
+git --version
+```
 
-Only core members can deploy the website.
+If Git is not installed, you can download it from the [official Git website](https://git-scm.com/install/).
 
-1. Clone the `pylonmc.github.io` repository: `git clone https://github.com/pylonmc/pylonmc.github.io`
-2. Deploy the site by running **in the pylonmc.github.io repository** `mkdocs gh-deploy --config-file ../pylon-docs/mkdocs.yml --remote-branch master`
```

Comment:
You removed it without providing a replacement, I'd still like this documented somewhere

Reply:
GitHub Actions now automatically deploys the site. I think there is no need to keep this now.